node/src/neighborhood/mod.rs

```diff
@@ -5515,6 +5685,11 @@ mod tests {
         let subject_addr = subject.start();
         let peer_actors = peer_actors_builder().ui_gateway(ui_gateway).build();
         subject_addr.try_send(BindMessage { peer_actors }).unwrap();
+        let pool_bind_message = pool_bind_message_builder()
+            .stream_handler_pool(make_recorder().0)
+            .dispatcher(make_recorder().0)
+            .build();
```

Comment:
These big structures of code repeat so much in all these tests that it'd be better to have a utility function for it. 
However, if you disable the subscribing system for streams to be shut down by directives from GossipAcceptor itself, you won't have this issue (with me) any longer or am I wrong? If it turns out staying the request means.

Reply:
Not sure what you meant by your second sentence, but I thought the first sentence was a good idea, so I did it.